lib/internal/modules/cjs/loader.js

```diff
@@ -839,6 +840,9 @@ var resolvedArgv;
 let hasPausedEntry = false;
 
 function wrapSafe(filename, content) {
+  // enable source-map tracking with --cache-source-maps.
```

Comment:
Is this a TODO?

Reply:
Yes, I would like to introduce a flag that enables source-map tracking ... I thought it might be worth having a comment that mentions that the function is a no-op unless that flag is set.